tests/feature/TransferTest.php

```diff
@@ -0,0 +1,77 @@
+<?php
+
+namespace Tests\Feature;
+
+use App\Transfer;
+use App\User;
+use App\Wallet;
+use Tests\TestCase;
+use Illuminate\Foundation\Testing\WithoutMiddleware;
+use Illuminate\Foundation\Testing\DatabaseMigrations;
+use Illuminate\Foundation\Testing\DatabaseTransactions;
+
+class TransferTest extends TestCase
+{
+    use DatabaseMigrations, WithoutMiddleware;
+
+    /**
+     * A send transfer test.
+     *
+     * @return void
+     */
+    public function testSendTransfer()
+    {
+        $this->seed();
+
+        $user = User::find(1);
+        $this->be($user);
+
+        $this->post('api/transfer/user/send', [
+            'receiver'  => [
```

Comment:
These are tests. That is why there should be inline comments. 
It is testing to see if there exists on receiver phone number or email field.